mkdocs.yml

```diff
@@ -0,0 +1,32 @@
+site_name: Project Name
+site_description: __package-description__
+site_author: __author-name__
+repo_url: https://github.com/graze/skeleton-project
+theme:
+  name: 'material'
+  custom_dir: theme
```

Comment:
in sprout, i changed this to `docs/theme`. And moved the `theme` directory underneath the `docs` directory.